homeassistant/config.py

```diff
@@ -775,26 +763,10 @@ def _recursive_merge(conf: Dict[str, Any], package: Dict[str, Any]) -> Union[boo
             continue
 
         try:
-            p_integration = await async_get_integration(hass, p_name)
-        except IntegrationNotFound:
-            continue
-
-        if (
-            not hass.config.skip_pip
-            and p_integration.requirements
-            and not await async_process_requirements(
-                hass, p_integration.domain, p_integration.requirements
-            )
-        ):
-            _LOGGER.error(
-                "Unable to install all requirements for %s.%s", domain, p_name
-            )
-            continue
-
-        try:
+            p_integration = await async_get_integration_with_requirements(hass, p_name)
             platform = p_integration.get_platform(domain)
-        except ImportError:
-            _LOGGER.exception("Failed to get platform %s.%s", domain, p_name)
+        except (RequirementsNotFound, IntegrationNotFound, ImportError):
+            _LOGGER.exception("Platform error: %s", domain)
```

Comment:
Now we're always logging the full exception, but for IntegrationNotFound we just need to log the error.

Reply:
Right, same goes for the requirementnotfound. It's just ImportError that needs it. I'll split the two